mailbox/api/src/main/java/org/apache/james/mailbox/MessageManager.java

```diff
@@ -117,6 +117,17 @@
      */
     Iterator<MessageUid> expunge(MessageRange set, MailboxSession mailboxSession) throws MailboxException;
 
+    /**
+     * Deletes messages
```

Comment:
maybe you can explain a bit what is the different between delete and expunge method,.

Reply:
:+1:  Maybe we should cover the side effects (events ... )